src/megatron/energon/flavors/webdataset/video_data.py

```diff
@@ -0,0 +1,14 @@
+import dataclasses
```

Comment:
Missing LICENSE header

Reply:
Note: There are more missing, e.g. in epath config. Added them all